closure/library/BUILD

```diff
@@ -193,6 +193,16 @@ filegroup(
     ],
 )
 
+closure_js_library(
+    name = "base_lib",
+    srcs = [
+        "@com_google_javascript_closure_library//:closure/goog/base.js",
+        "@com_google_javascript_closure_library//:closure/goog/transitionalforwarddeclarations.js",
+    ],
+    # Prevent circular dependencies
+    closure_library_base = [],
+)
```

Comment:
Have you considered the alternative where this dependency is automatically added to deps by the macros if no_closure_library is not set?
If you do that then we should be able to get rid of whole piping around _closure_library_base.
The negative part is whenever you define a new rule, you will need to do the same but it might worth reducing the complexity.

Reply:
I went with your other suggestion about creating a separate rule just for defining base.